src/applications/disability-benefits/526EZ/sass/disability-benefits.scss

```diff
@@ -145,3 +145,13 @@
 .async-display-widget-alert-box {
   margin-bottom: 1em;
 }
+
+// This seems duplicative
```

Comment:
of `va-address-block`? I believe that was the original recommendation to use here, perhaps there are some differences?

Reply:
Yeah, I saw `va-address-block` as well, but we shouldn't reuse it here. The name is misleading. Also, that particular one indents the div quite a ways, which is undesired.

We could rename that one and do the indenting with another class, but that seemed a little odd. Not a bad solution, though. On the other hand, this was a pretty small addition, so I could go either way. :man_shrugging: